### DISCUSSION TITLE
Delete Agent Bot

### DISCUSSION THREAD

**Discussion by hadupa** [chatwoot/chatwoot#9344] [Q&A]
Hello Everyone,

Been playing around with the app to figure out how to use it but now I plan on getting something serious set up. Problem is, I've got a bunch of Agent Bots I created via the Super_Admin Dashboard and I need to delete them.

I found one workaround in the SA Dashboard UI: If I create an account, assign an Agent Bot to that Account, then Destroy that Account, the Agent Bot is removed as well. Is this the only current technique to remove them? I've searched through Documentation and haven't found anything on how to Destroy an Agent Bot.

Thanks, great software!

**Comment by joaomachado-pg** [comment]
nada ?

**Comment by sojan-official** [comment]
We now have a UI in account settings to manage agent bots, the ability to delete system agent bots will be available on super admin in the upcoming release. 

<img width="1658" alt="image" src="https://github.com/user-attachments/assets/ed1a328a-8ba8-4c4b-9352-17dc87b23ac5" />
